### PR TITLE
Feature handshake endpoint

### PR DESCRIPTION
**Pull request by esl4m**

### CODE REVIEW

**Comment by HazemKhaled** [comment]
@esl4m check this please https://app.codacy.com/app/Knawat/knawat-dropshipping-opencart/pullRequest?prid=3426473

File: upload/system/library/knawat_dropshipping/knawatochandshake.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    /**
+     * Check Knawat cron is configured
+     */
+    public function knawatCronIsConfingured(){
+        //check for product cron time
+        $oldTime = $this->model_extension_module_knawat_dropshipping->get_knawat_meta('8162', 'time', 'cron_time' );
+        $current_time = time();
+        $difference = $current_time - (int)$oldTime;
+        if($difference > 10800){
```
Comment (hamusthekiller) [inline upload/system/library/knawat_dropshipping/knawatochandshake.php]: Those lines are confusing which is true and which are false to be able to confirm the cron job configuration.

File: upload/system/library/knawat_dropshipping/knawatochandshake.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+     */
+    public function __get($name)
+    {
+        return $this->registry->get($name);
+    }
+
+    /**
+     * Get OpenCart version
+     */
+    public function getOCVersion()
```
Comment (hamusthekiller) [inline upload/system/library/knawat_dropshipping/knawatochandshake.php]: Why don't you just return VERSION if u are not adding anything in the if condition?

**Comment by HazemKhaled** [comment]
no needed anymore

**Pull request closed without merging**